CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewTopComponent.java

```diff
@@ -85,6 +86,7 @@ public NotesViewTopComponent() {
             }
             notesViewPane.updateNotesUI();
         });
+        jfxContainer.setMinimumSize(new Dimension(1000, 500));
```

Comment:
A minimum width of 1000 is far too wide for a view pane.
Can we try a value around 400.

Reply:
As per my comment below, I'd actually prefer to avoid playing around the view sizes altogether. Rather the view should be made to be usable with the default size determined by the application